.github/workflows/workflow.yaml

```diff
@@ -0,0 +1,252 @@
+      - uses: convictional/trigger-workflow-and-wait@v1.3.0
+        with:
+          owner: kuzzleio
+          repo: documentation
+          github_token: ${{ secrets.ACCESS_TOKEN_CI }}
+          workflow_file_name: dead_links.workflow.yml
+          ref: ${{ steps.extract-refs.outputs.fw-branch }}
+          inputs: '{"repo_name": "${{ steps.extract-refs.outputs.repo }}", "branch": "${{ github.head_ref }}", "version": "${{ steps.extract-refs.outputs.version }}"}'
+
+  error-codes-check:
+    name: Documentation - Error codes check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/install-packages
+      - run: npm install
+      - name: Test error codes
+        run: ./.ci/scripts/check-error-codes-documentation.sh
+
+  lint:
+    name: Lint - Node.js LTS ${{ matrix.node-version }}
+    runs-on: ubuntu-latest
+    needs: [prepare-matrix]
+    strategy:
+      matrix: 
+        node-version: ${{ fromJson(needs.prepare-matrix.outputs.matrix).node-version }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/install-packages
+      - uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - uses: ./.github/actions/cache-node-modules
+        with:
+          NODE_VERSION: ${{ matrix.node-version }}
+      - uses: ./.github/actions/es-lint
+
+  unit-tests:
+    name: Unit Tests
+    needs: [lint, prepare-matrix]
+    strategy:
+      matrix:
+        node-version: ${{ fromJson(needs.prepare-matrix.outputs.matrix).node-version }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/install-packages
+      - uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - uses: ./.github/actions/cache-node-modules
+        with:
+          NODE_VERSION: ${{ matrix.node-version }}
+      - uses: ./.github/actions/unit-tests
+
+  build-and-run-kuzzle:
+    needs: [unit-tests, prepare-matrix]
+    name: Build and Run ${{ matrix.kuzzle-image }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: ${{ fromJson(needs.prepare-matrix.outputs.matrix).node-version }}
+        kuzzle-image: ['kuzzle']
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/install-packages
+      - uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - uses: ./.github/actions/cache-node-modules
+        with:
+          NODE_VERSION: ${{ matrix.node-version }}
+      - uses: ./.github/actions/build-and-run-kuzzle
+        with:
+          KUZZLE_IMAGE: ${{ matrix.kuzzle-image }}
+
+
+  functional-tests:
+    name: Functional Tests - ${{ matrix.test_set }} - Node.js v${{ matrix.node-version }}
+    needs: [unit-tests, prepare-matrix]
+    strategy:
+      matrix:
+        test_set: [http, websocket, "legacy:mqtt", "legacy:http", "legacy:websocket"]
```

Comment:
Mix of strings with `"` and without

Reply:
This is normal, Github Actions try to interpret the `:` as a YAML property, I don't know really why 😓